lib/src/cancelable_operation.dart

```diff
@@ -68,7 +68,10 @@ class CancelableOperation<T> {
   /// If this operation completes, this completes to the same result as [value].
   /// If this operation is cancelled, the returned future waits for the future
   /// returned by [cancel], then completes to [cancellationValue].
-  Future<T> valueOrCancellation([T cancellationValue]) {
+  ///
+  /// If [T] is not nullable then [cancellationValue] must provide a valid
```

Comment:
Should we be collecting any breaking changes we _should_ be making to improve ergonomics in an issue or something?

Reply:
https://github.com/dart-lang/async/issues/109